src/Media_Command.php

```diff
@@ -412,8 +418,13 @@ public function import( $args, $assoc_args = array() ) {
 				}
 			}
 
-			if ( Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
-				WP_CLI::line( $success );
+			if ( $field = Utils\get_flag_value( $assoc_args, 'porcelain' ) ) {
```

Comment:
Need to move the assignment out of the condition. I'd also suggest renaming `$field`.

```suggestion
			$porcelain = Utils\get_flag_value( $assoc_args, 'porcelain' );

			if ( $porcelain ) {
```

Reply:
Makes sense! New commit incoming.